civis_jupyter_notebooks/tests/test_notebook_config.py

```diff
@@ -48,28 +47,21 @@ def test_find_and_install_requirements_success(self, persistence_find_and_instal
 
     @patch('civis_jupyter_notebooks.platform_persistence.find_and_install_requirements')
     @patch('civis_jupyter_notebooks.platform_persistence.logger')
-    @patch('civis_jupyter_notebooks.log_utils.setup_file_logging')
-    def test_find_and_install_requirements_shows_errors(self, log_setup, logger, persistence_find_and_install_requirements):
-        mock_logger = MagicMock(logging.Logger)
+    def test_find_and_install_requirements_logs_errors(self, logger, persistence_find_and_install_requirements):
         persistence_find_and_install_requirements.side_effect = platform_persistence.NotebookManagementError('err')
-        log_setup.return_value = mock_logger
         c = Config({})
         notebook_config.find_and_install_requirements('path', c)
-        mock_logger.error.assert_called_with("Unable to install requirements.txt:\nerr")
-        assert(c.NotebookApp.default_url == log_utils.USER_LOGS_URL)
+        logger.error.assert_called_with("Unable to install requirements.txt:\nerr")
```

Comment:
Should this still be `mock_logger`? Or is this just coming from the `patch`?

Reply:
This is coming from the patch. We used to be testing the file logger here, which was the return value of the `log_setup` patched function, but now we're testing the stream logger, which comes from the `platform_persistence.logger` patch, which is itself a logger. (If that makes sense)